### DISCUSSION TITLE
Disable grouped notifications/reminders

### DISCUSSION THREAD

**Discussion by TheWranger** [iSoron/uhabits#1593] [Feature Requests]
Android (Oreo and newer, I believe) automatically groups app notifications together if there's three or more. This makes it hard to even notice if a new reminder has come through on the app, or how many have come through without opening the notification area manually. It gives much more urgency to respond to a habit reminder if there's several notifications piling up.

Of course, some people might find having so many notifications overwhelming, so it should be toggleable.

Apparently it is possible for developers to disable this on a per-app basis. Could this be implemented? Or if not, is there no workaround that could force this behaviour?

**Comment by iSoron** [comment]
Thank you for the feature request, @TheWranger. There are unofficial ways to prevent notifications from bundling together (specifically, assigning random/distinct notification groups) , but since this is an OS-level feature, I don't think we should be fighting against it. Android itself should make this configurable. I would only consider implementing this hack if there's significant user demand; so if anyone else would like to see it, please upvote.
